modules/osdk-cli-reference-cleanup.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-cli-reference.adoc
+
+[id="osdk-cli-reference-cleanup_{context}"]
+= cleanup
+
+The `operator-sdk cleanup` command destroys and removes resources that were created for an Operator that was deployed with the `run` command.
+
+== packagemanifests
+
+`cleanup packagemanifests` subcommand destroys an Operator that was deployed with OLM by using the `run packagemanifests` command.
+
+.`packagemanifests` arguments
+[options="header",cols="1,3"]
+|===
+|Arguments |Description
+
+|`--include` (string)
+|The file path to Kubernetes resource manifests, such as Role objects and Subscription objects. These supplement or override defaults generated by `run` or `cleanup`.
```

Comment:
Please revise "Role objects and Subscription objects" to follow either the general reference formatting guidelines or as object references.  See the [API object formatting](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting) section of the doc guidelines.

Because the context does not name a specific role or subscription, my sense is that a general reference is appropriate.  My summary of the guidelines for a general reference: all lower case, no backticks, no need to follow with "resource" or "object".  Maybe:

"The file path to the Kubernetes resource manifests, such as role and subscription objects."